Vagrantfile

```diff
@@ -6,72 +6,21 @@
 # backwards compatibility). Please don't change it unless you know what
 # you're doing.
 
+$script = <<SCRIPT
```

Comment:
Ah, I see, thanks for explaining!

Would you mind adding a comment pointing to the SO link, and/or giving the term to google? I think that would be useful for others who read this file in the future.